### PR TITLE
Added npm-set registry command to installation file

### PR DESCRIPTION
**Pull request by JayaKrishnaNamburu**
Hey @juanpicado here is the change that I made to the docs, and this is how it looks after render
Screenshot 2019-04-16 at 11 25 51 AM

Please let me know if any changes or words that need to be changed :)

### CODE REVIEW

**Comment by JayaKrishnaNamburu** [comment]
Fixes #63

File: docs/install.md [hunk 1]
```diff
@@ -46,6 +46,17 @@ warn --- http address - http://localhost:4873/ - verdaccio/3.0.0
 
 For more information about the CLI, please [read the cli section](cli.md).
 
+You can set the registry by using the following command.
+
+```bash
+npm set registry http://localhost:4873/
+```
+or you can pass a ``--registry`` flag when needed.
```
Comment (juanpicado) [inline docs/install.md]: Please leave a blank line between lines 53 and 54, crowdin is quite picky with code blocks that collides with text.

**Comment by JayaKrishnaNamburu** [comment]
Hey @juanpicado can you check this now!

**Comment by juanpicado** [comment]
Thank u @JayaKrishnaNamburu :-) welcome to the Verdaccio world.

**Comment by JayaKrishnaNamburu** [comment]
> Thank u @JayaKrishnaNamburu :-) welcome to the Verdaccio world.

Thank you and wishes on the 3rd Anniversary 😄

**Pull request merged**